themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties

```diff
@@ -0,0 +1,224 @@
+unknownLoginRequesterMessage=De login requester is onbekend
+loginRequesterNotEnabledMessage=De login requester is niet geactiveerd
+bearerOnlyMessage=Bearer-only applicaties mogen geen browser login initiëren
+standardFlowDisabledMessage=Client is mag geen browserlogin starten met het opgegeven response_type. Standard flow is uitgeschakeld voor de client.
+implicitFlowDisabledMessage=Client is mag geen browserlogin starten met opgegeven response_type. Implicit flow is uitgeschakeld voor de klant.
+invalidRedirectUriMessage=Ongeldige omleiding uri
+unsupportedNameIdFormatMessage=Niet-ondersteunde NameIDFormat
+invalidRequesterMessage=Ongeldige requester
+registrationNotAllowedMessage=Registratie is niet toegestaan
+resetCredentialNotAllowedMessage=Het resetten van de aanmeldgegevens is niet toegestaan
+permissionNotApprovedMessage=Toestemming niet goedgekeurd.
+noRelayStateInResponseMessage=Geen relay state in antwoord van de identity provider.
+insufficientPermissionMessage=Onvoldoende rechten om identiteiten te koppelen.
+couldNotProceedWithAuthenticationRequestMessage=Het authenticatie verzoek naar de identity provider wordt afgebroken.
+couldNotObtainTokenMessage=Kan het token van identity provider niet ophalen.
+unexpectedErrorRetrievingTokenMessage=Onverwachte fout bij het ophalen van de token van de identity provider.
+unexpectedErrorHandlingResponseMessage=Onverwachte fout bij het verwerken van de response van de identity provider.
+identityProviderAuthenticationFailedMessage=Verificatie mislukt. Er kon niet worden geauthenticeerd met de identity provider.
+identityProviderDifferentUserMessage=U bent geauthenticeerd als {0}, maar u werd als {1} verwacht geauthenticeerd te zijn
+couldNotSendAuthenticationRequestMessage=Kan het authenticatie verzoek niet verzenden naar de identity provider.
+unexpectedErrorHandlingRequestMessage=Onverwachte fout bij het verwerken van het authenticatie verzoek naar de identity provider.
+invalidAccessCodeMessage=Ongeldige toegangscode.
+sessionNotActiveMessage=Sessie niet actief.
+invalidCodeMessage=Er is een fout opgetreden, log daarom opnieuw in.
+identityProviderUnexpectedErrorMessage=Onverwachte fout tijdens de authenticatie met de identity provider
+identityProviderNotFoundMessage=Kan geen identity provider vinden die aan de identifier voldoet.
+identityProviderLinkSuccess=Uw account is met succes gekoppeld aan {0} account {1}.
+staleCodeMessage=Deze pagina is niet langer geldig. Ga terug naar uw applicatie en log opnieuw in, alstublieft
+realmSupportsNoCredentialsMessage=Realm ondersteunt geen enkel soort aanmeldgegeven.
+identityProviderNotUniqueMessage=Realm ondersteunt meerdere identity providers. Er kon niet bepaald worden welke identity provider er gebruikt zou moeten worden tijdens de authenticatie.
+emailVerifiedMessage=Uw e-mailadres is geverifieerd.
+staleEmailVerificationLink=De link die u gebruikt is verlopen en daarom niet langer geldig. Misschien heeft u uw uw e-mailadres al geverifieerd?
```

Comment:
"De link die u gebruikt is niet langer geldig, wellicht omdat u uw e-mailadres al eerder geverifieerd heeft."